src/ramprakash/narcissisticNumbers.ts

```diff
@@ -0,0 +1,38 @@
+export const isNarcissistic = (num: number): boolean => {
+  if (num <= 0) {
+    return false;
+  }
+
+  if(num > 0 && num < 10) {
```

Comment:
Wrong conditional. It should be ||

Reply:
Shouldn't be `||` too, that is dangerous. This condition should be just `num < 10` as negative numbers are already filtered just above.